crates/bevy_pbr/src/light.rs

```diff
@@ -92,6 +92,9 @@ impl Default for PointLightShadowMap {
 /// approximation for light sources VERY far away, like the sun or
 /// the moon.
 ///
+/// The light's default direction is negative-Z (0, 0, -1) and is
+/// adjusted by the GlobalTransform.
```

Comment:
```suggestion
/// adjusted by the [`GlobalTransform`] of the entity that has this component.
```

Reply:
I'd rather remove the mention of `GlobalTransform` as it may make people think they should edit it instead of `Transform`.